packages/polaris-viz/src/components/DonutChart/Chart.tsx

```diff
@@ -79,15 +80,24 @@ export function Chart({
   const [activeIndex, setActiveIndex] = useState<number>(-1);
   const selectedTheme = useTheme();
 
-  const seriesCount = clamp({amount: data.length, min: 1, max: Infinity});
+  const seriesCount = clamp({
+    amount: data.length,
+    min: 1,
+    max: DONUT_CHART_MAX_SERIES_COUNT,
+  });
+
+  const seriesData = data
+    .slice(0, seriesCount)
+    .filter(({data}) => Number(data[0]?.value) > 0);
```

Comment:
Hey, I don't have a ton of context on the `DonutChart` so I'll defer to whatever you think is best!

Reply:
I think as a consumer it would be odd to assume that the data should be sorted. Passing in some data and then having truncated/not shown would be inaccurate and confusing. Since we are now showing a limited amount of items in the chart now (5) I think it makes sense to filter, sort and truncate the data. I'll go ahead and make that change 👍